mgmtsystem_action/views/mgmtsystem_action.xml

```diff
@@ -0,0 +1,337 @@
+                        <group name="meta">
+                            <field name="reference" />
+                            <field
+                                name="tag_ids"
+                                widget="many2many_tags"
+                                options="{'color_field': 'color', 'no_create_edit': True}"
+                            />
+                        </group>
+                    </group>
+                    <notebook>
+                        <page name="desc" string="Description">
+                            <field name="description" nolabel="1" />
+                        </page>
+                        <page name="extra" string="Extra Information">
+                            <group name="main_extra">
+                                <group name="extra">
+                                    <field name="sequence" />
+                                    <field name="system_id" />
+                                    <field
+                                        name="company_id"
+                                        groups="base.group_multi_company"
+                                    />
+                                </group>
+                                <group name="dates">
+                                    <field name="create_date" />
+                                    <field
+                                        name="date_open"
+                                        invisible="date_open == False"
+                                    />
+                                    <field
+                                        name="date_closed"
+                                        invisible="date_closed == False"
+                                    />
+                                </group>
+                            </group>
+                        </page>
+                    </notebook>
+                </sheet>
+                <chatter />
+            </form>
+        </field>
+    </record>
+
+    <record id="view_mgmtsystem_action_kanban" model="ir.ui.view">
+        <field name="name">mgmtsystem.action.kanban</field>
+        <field name="model">mgmtsystem.action</field>
+        <field name="arch" type="xml">
+            <kanban default_group_by="stage_id" class="o_kanban_small_column">
+                <field name="name" />
+                <field name="priority" />
+                <field name="tag_ids" />
+                <field name="sequence" />
+                <field name="type_action" />
+                <field name="reference" />
+                <field name="user_id" />
+                <field name="create_date" />
+                <field name="date_deadline" />
+                <field name="activity_ids" />
+                <field name="activity_state" />
+                <field name="message_needaction_counter" />
+                <templates>
+                    <field name="date_deadline" />
+                    <t t-name="kanban-box">
```

Comment:
@Kimkhoi3010  This line is preventing this module to be merged, could you fix it please
https://github.com/OCA/management-system/actions/runs/11903685874/job/33171125454#step:8:92